src/applications/static-pages/authenticated-homepage/components/App/index.js

```diff
@@ -0,0 +1,34 @@
+// Node modules.
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { selectShowAuthenticatedHomepage } from 'applications/static-pages/authenticated-homepage/selectors';
+
+export const App = ({ showAuthHomePage }) => {
+  if (!showAuthHomePage) {
+    return null;
+  }
+
+  document.getElementById('homepage').style.display = 'none';
+  return <div>This is authenticated homepage extreme!</div>;
+};
+
+App.propTypes = {
+  showAuthHomePage: PropTypes.bool,
+};
+
+const mapStateToProps = state => {
+  const loggedIn = state.user.login.currentlyLoggedIn;
+  const authHomepageFF = selectShowAuthenticatedHomepage(state);
+  const authHomepageLS = localStorage.getItem('AUTHENTICATED_HOMEPAGE');
+
+  return {
+    showAuthHomePage: loggedIn && (authHomepageFF || authHomepageLS),
```

Comment:
cool for now, but we might want to be able to turn this feature off via localstorage even if the feature flag is on, like we do for Profile 2.0

Reply:
You're certainly right! Updated 💯